docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV PRESTO_HOME="/opt/presto-server"
 COPY $PRESTO_PKG .
 COPY $PRESTO_CLI_JAR /opt/presto-cli
 
-RUN dnf install -y java-11-openjdk less procps python3 \
+RUN dnf install -y java-17-openjdk less procps python3 \
```

Comment:
Is this gonna be 17.0.13?

Reply:
It will be whichever version the stream9 maintainers have in the repository. I don't think we will require a minimum/maximum version of the 17 JDK.